deepchem/data/datasets.py

```diff
@@ -526,9 +526,8 @@ def get_data_shape(self):
     if not len(self.metadata_df):
       raise ValueError("No data in dataset.")
     sample_X = load_from_disk(
-        os.path.join(self.data_dir, next(self.metadata_df.iterrows())[1]['X']))[
-            0]
-    return np.shape(sample_X)
+        os.path.join(self.data_dir, next(self.metadata_df.iterrows())[1]['X']))
+    return np.shape(sample_X)[1:]
 
```

Comment:
Ah, I see what's happening here. Ok, this makes sense to (using the `X` shape directly rather than that of the first element)